### PR TITLE
Topic8 Download Links

### PR DESCRIPTION
**Pull request by jameshughes89**
Download links for `LinkedStack` and `LinkedStackTest` added to the end of the Topic8 page.

### CODE REVIEW

**Comment by twentylemon** [comment]
Moved to draft, there's code copied from #76

**Comment by jameshughes89** [comment]
Yuck. 

I'll fix tomorrow

**Comment by jameshughes89** [comment]
I guess I ended up including the copy/paste from `ArrayStack` playing to `LinkedStack`? Unclear on where that happened, but whatever.

File: src/site/topic8.rst [hunk 1]
```diff
@@ -274,8 +274,83 @@ Introduction Errors for Fun
...
+* We only need to make one change
+    * ``ArrayStack`` -> ``LinkedStack``
+
+.. code-block:: java
+    :linenos:
+    
+    // Create an ArrayStack
+    Stack<Integer> myStack = new LinkedStack<Integer>();
+
+    // Check stack is empty
```
Comment (twentylemon) [inline src/site/topic8.rst:290]: Again semi related to #46, but you could include these as actual main classes in the repo; like, actual java source files.

**Review by twentylemon — APPROVED** [review]

File: src/site/topic8.rst (hunk 1 above)
Comment (jameshughes89) [reply src/site/topic8.rst:290]: Grand idea. I'll create a suite of these.

**Pull request merged**